lib/fuzzy-finder-view.coffee

```diff
@@ -30,6 +30,16 @@ class FuzzyFinderView extends SelectListView
   getFilterKey: ->
     'projectRelativePath'
 
+  cancel: ->
+    if atom.config.get('fuzzy-finder.preserveLastSearch')
+      lastSearch = @getFilterQuery()
```

Comment:
calling `@getFilterQuery()` returns the filter text without a trailing line number such so `pack:8` would become `pack`.

Did you want to preserve this info across opens? A trailing `:NUMBER` allows you to open a file to a specific line.